tests/osaka/eip7692_eof_v1/eip7069_extcall/test_gas.py

```diff
@@ -185,3 +186,34 @@ def test_transfer_gas_is_cleared(
         + push_gas,
         out_of_gas_testing=False,
     )
+
+
+@pytest.mark.parametrize("opcode", [Op.EXTCALL, Op.EXTDELEGATECALL, Op.EXTSTATICCALL])
+def test_late_account_create(
+    state_test: StateTestFiller,
+    pre: Alloc,
+    state_env: Environment,
+    opcode: Op,
+):
+    """
+    Test EXTCALL to a non-existent account after another EXT*CALL has called it and not created it.
```

Comment:
```suggestion
    Test EXTCALL to a non-existent account after another EXT*CALL has called it and not
    created it.
```
New formatter ([ruff](https://docs.astral.sh/ruff/)) will complain about single-line docstrings.